CONTRIBUTING.md

```diff
@@ -66,6 +66,18 @@ If you would like to add new capabilities to the product as described in **B)**,
 
     Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.
 
+## Get To Know System App
+
+There are several chapters on Microsoft Learn that explains several aspects of the System app.  
+Here are some:
```

Comment:
It's twice "several", and instead of "some", you list all of them. That's why I suggest:
```suggestion
Before updating the System Application, familiarize yourself with the following sites on Microsoft Learn:
```